src/main/java/com/gmail/goosius/siegewar/enums/SiegeStatus.java

```diff
@@ -72,6 +72,35 @@ public String getName() {
 		return this.name;
 	}
 
+	public boolean allowsPlundering() {
+		return attackersWon();
+	}
+
```

Comment:
❔ 
- Should a method called getPlunderReductionPercentage() be in this class?

- Examples
  - With decisive attacker win, plunder is reduced by 0%
  - With defender surrender, plunder is reduced by 0%
  - With close attacker win, plunder is reduced by 50%

- Theoretically, in future some server might want to add plunder on close-defender-win, or reduce it on defender surrender, so if this method existed, it would be easier to add such support.

Reply:
This one I think doesn't get any simpler than it already is, it would just move code somewhere else.